nilearn/interfaces/fmriprep/load_confounds.py

```diff
@@ -290,34 +294,51 @@ def load_confounds(img_files,
     return confounds_out, sample_mask_out
 
 
-def _load_single(confounds_raw, strategy, demean, **kargs):
+def _load_confounds_for_single_image_file(image_file, strategy, demean, **kwargs):
     """Load confounds for a single image file."""
-    # Convert tsv file to pandas dataframe
-    # check if relevant imaging files are present according to the strategy
-    flag_acompcor = ("compcor" in strategy) and (
-        "anat" in kargs.get("compcor")
-    )
+    # Check if ica_aroma is in strategy, this will change which image_file is required
     flag_full_aroma = ("ica_aroma" in strategy) and (
-        kargs.get("ica_aroma") == "full"
+            kwargs.get("ica_aroma") == "full"
     )
-    confounds_raw, meta_json = _confounds_to_df(
-        confounds_raw, flag_acompcor, flag_full_aroma
+
+    confounds_file = _get_confounds_file(image_file, flag_full_aroma=flag_full_aroma)
+    confounds_json_file = _get_json(confounds_file)
+
+    return _load_single_confounds_file(confounds_file=confounds_file,
+                                       confounds_json_file=confounds_json_file,
```

Comment:
Could you swap the arguments such that the order match the one of the signature of the function? It'd be much easier to read